tests/functions/test_llama.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+
+import pixeltable as pxt
+
+# from ..conftest import DO_RERUN
+from ..utils import skip_test_if_no_client, skip_test_if_not_installed, validate_update_status
+
+
+@pytest.mark.remote_api
+# @pytest.mark.flaky(reruns=3, reruns_delay=8, condition=DO_RERUN) # Add back if needed
+class TestLlama:
+    def test_chat_completions(self, reset_db: None) -> None:
+        skip_test_if_not_installed('openai')
+        skip_test_if_no_client('llama')
+        t = pxt.create_table('test_llama_tbl', {'input': pxt.String})
+        from pixeltable.functions.llama import chat_completions
+
+        # Use a small, fast model available via the API for testing
+        # Note: Model availability might change, adjust if needed.
+        # Llama-3.1-8B-Instruct seems like a reasonable choice if available.
+        # Using Llama-3.3-8B-Instruct as per the example in the docs.
+        test_model = 'Llama-4-Scout-17B-16E-Instruct-FP8'
```

Comment:
The code comment says "Using Llama-3.3-8B-Instruct" but the actual code says otherwise.